pkg/components/master.go

```diff
@@ -174,6 +191,16 @@ func (m *master) createInitScript() string {
 	return strings.Join(script, "\n")
 }
 
+func (m *master) createExitReadOnlyScript() string {
+	script := []string{
+		initJobWithNativeDriverPrologue(),
+		// TODO: remove || in the future
+		"/usr/bin/yt execute master_exit_read_only '{}' || echo 'master_exit_read_only is supported since 23.2'",
```

Comment:
We have a nice conventions for the compatibility code that must be erased in some future. It must be prepended with a comment of form `COMPAT(username): remove something when something happens`. The important part is to specify explicitly, what condition must hold in order to remove this COMPAT; this allows code maintainers to periodically grep codebase for the word "COMPAT" and easily understand, which COMPATs may be safely removed without trying to remember the details of the code 6 months ago.

In your case I'd write
```
// COMPAT(l0kix2): remove || part when the compatibility with 23.1 and older is dropped.
```
(also, mind the dot in the end of the line)